app/views/proposal/index.html.haml

```diff
@@ -97,5 +97,5 @@
                   class: 'btn btn-default', id: "edit_proposal_#{event.id}"
   .row
     .col-md-12
-      - if @conference.cfp_open? || (current_user.has_role? :organizer, @conference)
+      - if @conference.cfp_open?
```

Comment:
I know! But as an Organizer I want to be able to submit proposals after the CFP is closed. Not sure if we already cover this in the admin menu.

@hennevogel @differentreality what do you think?